lua/smart-splits/utils.lua

```diff
@@ -1,5 +1,11 @@
 local M = {}
 
+local function checkIfWSL()
+  return io.popen('grep -i WSL /proc/version')
+end
+
+M.isWSL = checkIfWSL()
+
```

Comment:
We can also just do this:

```suggestion
function M.is_wsl()
  return vim.env.WSL_DISTRO_NAME ~= nil and vim.env.WSL_DISTRO_NAME ~= ""
end
```

Reply:
I found this via a quick web search, and this way we don't need to shell out via `io.popen`